diff --git a/application/views/login.php b/application/views/login.php
index d3c5a72..820c0f6 100644
--- a/application/views/login.php
+++ b/application/views/login.php
@@ -26,7 +26,7 @@
 				<div id="login_form">
 					<div class="input-group">
 						<span class="input-group-addon input-sm"><span class="glyphicon glyphicon-user"></span></span>
-						<input class="form-control" placeholder="<?php echo $this->lang->line('login_username')?>" name="username" type="username" size=20 autofocus></input>
+						<input class="form-control" placeholder="<?php echo $this->lang->line('login_username')?>" name="username" type="text" size=20 autofocus></input>
 					</div>
 
 					<div class="input-group">
diff --git a/public/css/login.css b/public/css/login.css
index 723de4b..c2acce6 100644
--- a/public/css/login.css
+++ b/public/css/login.css
@@ -36,7 +36,7 @@ form
 	padding:6% 4%;
 }
 
-input[type="username"], input[type="password"]
+input[type="text"], input[type="password"]
 {
 	width:100%;
 	margin-bottom:4%;
