diff --git a/application/helpers/locale_helper.php b/application/helpers/locale_helper.php
index 2358f0e0..2c33e09f 100644
--- a/application/helpers/locale_helper.php
+++ b/application/helpers/locale_helper.php
@@ -278,6 +278,12 @@ function get_payment_options()
 		$payments[$lang->line('sales_credit')] = $lang->line('sales_credit');
 	}
 
+        $payments[$lang->line('sales_credit')] = [
+                'Visa' => 'Visa',
+                'Mastercard' => 'Mastercard',
+                'American Express' => 'American Express',
+        ];
+
 	$payments[$lang->line('sales_due')] = $lang->line('sales_due');
 	$payments[$lang->line('sales_check')] = $lang->line('sales_check');
 
